modules/serverless-cluster-sizing-requirements.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+
+[id="cluster-size-requirements_context"]
+= Cluster size requirements
+
+Determine how many nodes and pods you require for your OpenShift Container Platform cluster. Cluster scalability correlates to the number of pods in a cluster environment. That number influences the other numbers in your setup.
```

Comment:
"Cluster scalability correlates to the number of pods in a cluster environment. That number influences the other numbers in your setup." I don't think it's clear enough what this means. Please look at rewriting this sentence so it explains better.